meson.build

```diff
@@ -113,7 +113,7 @@ pipewire_dep = dependency('libpipewire-0.3',
                           required: get_option('pipewire'))
 libsystemd_dep = dependency('libsystemd', required: get_option('systemd'))
 
-bwrap = find_program('bwrap')
+bwrap = find_program('bwrap', required: false)
```

Comment:
I wonder if we should make `bwrap` required on Linux, so that it isn't mistakenly leaved out if you compile yourself.